mars/api.py

```diff
@@ -162,8 +162,14 @@ def get_graph_state(self, session_id, graph_key):
             state = state_obj.value if state_obj else 'preparing'
         except ActorNotExist:
             raise GraphNotExists
-        state = GraphState(state.lower())
-        return state
+        return GraphState(state.lower())
+
+    def get_graph_traceback(self, session_id, graph_key):
```

Comment:
Maybe ``get_graph_exc_info`` is better, as exc_info contains exc_type, exc and traceback.

Reply:
Fixed.